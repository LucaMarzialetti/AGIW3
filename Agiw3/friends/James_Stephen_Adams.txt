--url_fb https://www.facebook.com/jamesstephen.adams
--friends
VM16615:3 Earl A. Von Buck
VM16615:3 Lorraine Wright Stewart
VM16615:3 Carolyn Secrease Wick
VM16615:3 Michael Pate
VM16615:3 Ronnie Kamps
VM16615:3 Kathryn Engle
VM16615:3 Janie Bailey
VM16615:3 Jim Triplett
VM16615:3 Kimberly King
VM16615:3 Kelly Marapao
VM16615:3 Melanie Ohlstrom-Herrera
VM16615:3 Sandra Bryant
VM16615:3 Pete Wick
VM16615:3 Kim King
VM16615:3 Phillip Tennell
VM16615:3 David Figg
VM16615:3 William Groeneveld
VM16615:3 Marilyn Wright
VM16615:3 Mary Evans
VM16615:3 Christi Lee-Sebern
VM16615:3 Helen Martin
VM16615:3 Marcus Raya
VM16615:3 Doug Forker
VM16615:3 Gerry Romos
VM16615:3 John Rogalla
VM16615:3 Jackie Taylor Stiegler
VM16615:3 Al Flores
VM16615:3 Noreen Down Lees
VM16615:3 Barbara B. Fisher
VM16615:3 Lisa Hanson Delaney
VM16615:3 Suzanne Carpenter
VM16615:3 Brian Reed
VM16615:3 Joi Ann Buck
VM16615:3 Jon Wishard
VM16615:3 Kathy Manis Hooper
VM16615:3 Dawn Clark
VM16615:3 Jacquelyn Mccormick
VM16615:3 Jack Borges
VM16615:3 Carmel Lynn Larimer Martin
VM16615:3 Carol S. Chilcoat-Roever
VM16615:3 Sherrie Atteberry
VM16615:3 Dacia Frazer Von Buck
VM16615:3 Shannon Eckroth
VM16615:3 Kimberly Timmerman Cones
VM16615:3 David Hovde
VM16615:3 Karinda Rae
VM16615:3 Wayne Deja
VM16615:3 Elizabeth Aloe
VM16615:3 Sue Whitcomb LaCoe
VM16615:3 Bobbi Killgore
VM16615:3 H Jay Gee
VM16615:3 Jeanine McNamee Thiebaud
VM16615:3 Tony Agrusa
VM16615:3 Gina Killingsworth
VM16615:3 Traci Harrington
VM16615:3 Mark Calsing
VM16615:3 Beverly Goble
VM16615:3 Rita Rowe
VM16615:3 April Caligari
VM16615:3 Chris Dieterich Pehl
VM16615:3 Jessica Yates
VM16615:3 Lorrel Swann-Spencer
VM16615:3 Lisa Holland Gillem
VM16615:3 Therese T Therese
VM16615:3 Vanessa Henke
VM16615:3 Liz Jendros
VM16615:3 Toni Ford
VM16615:3 Annette Neer
VM16615:3 Lloyd Lee
VM16615:3 Terri Fisher
VM16615:3 Scott Frazier
VM16615:3 Frank Morin
VM16615:3 Nicole Koning
VM16615:3 Brandy Stevens
VM16615:3 Priscilla Barnes
VM16615:3 Debbie Blankenship
VM16615:3 Michelle Webster
VM16615:3 Jody Johnson
VM16615:3 Dora Richard
VM16615:3 Matthew Tauanuu
VM16615:3 Richard Bailey
VM16615:3 Dale Adams
VM16615:3 Chris Andersen
VM16615:3 Bruce Hensley
VM16615:3 Kielo Smith
VM16615:3 Debbie Stone Blankenship
VM16615:3 Peter Malouf
VM16615:3 John Anderson
VM16615:3 Ron Potts
VM16615:3 Gregg Chamberlain
VM16615:3 Mark Biallas
VM16615:3 Roger Cason
VM16615:3 Maryan Infield
VM16615:3 Robert Wallace
VM16615:3 David Malouf
VM16615:3 Daniel Malouf
VM16615:3 Jaylene Hutchison
VM16615:3 Mark England
VM16615:3 Kathryn Aceto
VM16615:3 Chris Blagg
VM16615:3 Roscoe Brandenburg
VM16615:3 Rick Wyatt
VM16615:3 Jackie Gilbert
VM16615:3 Cheri Wulff Lucas
VM16615:3 John Lindsey
VM16615:3 Annie Laurie
VM16615:3 Ashley Stone
VM16615:3 Amy Austin
VM16615:3 Rebecca Sloan
VM16615:3 Connie Yanez
VM16615:3 Amber Stone
VM16615:3 Gary Keuntje
VM16615:3 Patty Hayes
VM16615:3 Karen Newsom
VM16615:3 Terrie Vieira
VM16615:3 Pam Dean Forsythe
VM16615:3 Barbara Sinor
VM16615:3 Teresa Hughes
VM16615:3 Peggy McHale
VM16615:3 Lisa Hall Nadalsky
VM16615:3 Charlene Lozito
VM16615:3 Bill Reynolds
VM16615:3 Nikki Reinhart Diaz
VM16615:3 Lois Rezler
VM16615:3 Sheila Farley
VM16615:3 Diana DeGarmo
VM16615:3 Debbie Kaminski
VM16615:3 Lindi Woods
VM16615:3 Julie Thompson
VM16615:3 Lori Sylvia
VM16615:3 Ronald Queen
VM16615:3 Lyle Bertholf
VM16615:3 Brian Shorter
VM16615:3 Mike Johnson
VM16615:3 Christelle Speaks
VM16615:3 Valerie Sendejas Moreno
VM16615:3 Juan Busby
VM16615:3 Barbara Horton Hoyt
VM16615:3 Mark Nicodemus
VM16615:3 Vicki Book
VM16615:3 Shelly Thompson
VM16615:3 Laurie Kay Riedel
VM16615:3 Suzanne McEachern
VM16615:3 Gary Elem
VM16615:3 Stephen Lamb
VM16615:3 Mike Howell
VM16615:3 Daniel Allen Umphenour
VM16615:3 Damon Reynoso
VM16615:3 Ed Russell
VM16615:3 Hope Hanselman
VM16615:3 Paso Robles
VM16615:3 Greg Stumph
VM16615:3 Jeanette Trompeter
VM16615:3 Robert Moore
VM16615:3 Tommy De La Rosa
VM16615:3 Allen Griswold
VM16615:3 Jenny Keel
VM16615:3 Susan Miller
VM16615:3 Chuck Barber
VM16615:3 Arthur Mooneyhan
VM16615:3 Bart Floercky
VM16615:3 Glen Starkey
VM16615:3 Stacey Wideman
VM16615:3 Royce Haislip
VM16615:3 Cindy Stoira
VM16615:3 Janet Amarillas
VM16615:3 Dave Grapentine
VM16615:3 Sabrina Johnson
VM16615:3 Anne Stewart
VM16615:3 Marlyce Dubsky Franks
VM16615:3 Dave Congalton
VM16615:3 C.j. Mo Mohammed
VM16615:3 Chuck Bruhn
VM16615:3 Mary Munoz
VM16615:3 Dale Evers
VM16615:3 Mark Stanton
VM16615:3 Green Flash
VM16615:3 Tania Ryan
VM16615:3 Marcy Almaguer
VM16615:3 Mike Kubel
VM16615:3 Donald N Jeanne Ikeler
VM16615:3 Leslie Burton
VM16615:3 Jennifer Moore Craytor
VM16615:3 Don England
VM16615:3 Tyrone Leekins
VM16615:3 LaVonna Lacy
VM16615:3 Anna Agostino Johnson
VM16615:3 Ramona Clayton
VM16615:3 Robert Becker
VM16615:3 Charlie J Mohammed
VM16615:3 Charlene Boyd Padgett