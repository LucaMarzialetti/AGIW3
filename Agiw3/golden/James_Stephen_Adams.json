{
    "PATTERN": {
        "tel": ["(903) 757-3311"],
        "addr": ["320 E. Methvin St. Longview",
        "TX 75601"],
        "email": []
    },
    "NER": {
        "LOC": [
            "Longview",
            "Methvin St. Longview",
            "Rosewood Park"
        ],
        "ORG": ["Rader Funeral Home",
            "Facebook",
            "Twitter"],
        "PER": [
            "James Stephen Adams",
            "Cass Gregg Harrison Marion Morris Panola Rusk",
            "James Stephen",
            "Steve Adams",
            "Editor"
        ]
    },
    "url": "https://www.news-journal.com/obituaries/2010/jun/18/james-stephen-adams/"
}